### ISSUE TITLE
SaveMix modal (savemixpopup) should go away when the user hits 'submit'

### ISSUE THREAD

**Issue by docsmage** [voiceofrae/RelaxIO#43]

**Comment by docsmage** [comment]
I've investigated this issue extensively, it seems to be a known bug in Angular 1.4 as documented in this thread: https://github.com/angular-ui/bootstrap/issues/3633

Here is the example for open(), close() and dismiss() given in the documentation: https://angular-ui.github.io/bootstrap/#/modal I attempted to adapt the example by adding ModalInstanceCtrl, but was ultimately not able to make it work. As of now the closeModal function in NavCtrl does nothing. This issue may be beyond my current abilities as a developer.

If an aspirational angular dev wants to get in and submit a potential resolution for this issue, I'm all for it. As of now I'm removing this issue from release 1.1. Also opened Issue #53 regarding the case with signuppopup, wherein it doesn't change if you click the links within the modal to change routes.
